protocol/messenger.go

```diff
@@ -4138,6 +4138,10 @@ func (m *Messenger) DeleteMessagesByChatID(id string) error {
 	return m.persistence.DeleteMessagesByChatID(id)
 }
 
+func (m *Messenger) MarkMessageAsUnread(chatID string, messageID string) (uint64, uint64, error) {
+	return m.persistence.MarkMessageAsUnread(chatID, messageID)
```

Comment:
Should we update `m.allChats` here as done in `markMessagesSeenImpl`?
https://github.com/status-im/status-go/blob/6197ed1bdb4b14ec7454e34086bafb2cf0f4952c/protocol/messenger.go#L4139-L4143